supervisor/backups/manager.py

```diff
@@ -0,0 +1,399 @@
+"""Backup manager."""
```

Comment:
We started to call this `module.py`. However, I'm fine to call this `manager.py` or `core.py`, we should just rename the other exists one. Maybe `core.py` would be a better fit? (I never liked `module.py`, but as you bring that on topic, we can discuss it)

Reply:
Yeah, was thinking about using "module" first, but the class was `BackupManager`  so `manager` felt more correct here, I don't really have strong feelings about it

Reply:
Let's rename it to `core.py`. I did the same now on os agent. I can later rename the others too

Reply:
"core" might be confusing with core which we use for homeassistant
"manager", "base", "module", "main", "element", "entry" are all better than "core" for this purpose IMO